lms/djangoapps/django_comment_client/management/commands/tests/test_rename_user.py

```diff
@@ -0,0 +1,128 @@
+'''
+Unit Tests for the manage.py command rename_user
+'''
+from django.core.management.base import CommandError
+import unittest
+from django.contrib.auth.models import User
+from pymongo import MongoClient
+from student.tests.factories import UserFactory
+import datetime
+from .. import rename_user
+import sys
+from StringIO import StringIO
+
+
+class TestUsernameRenameCommand(unittest.TestCase):
+    '''
+    Class which contains various unit tests for the manage.py rename_user command line tool
```

Comment:
We aim to make documentation as concise as possible; we should avoid
repeating anything that's already explicit in the code.

Here, it's obvious that this is a class of unittests because, well, it
defines a class that inherits from `unittest.TestCase`. We can shorten
this to:
```python
class TestUsernameRenameCommand(unittest.TestCase):
    """
    Verify functionality of `rename_user` management command
    """
```